test/image_dart_compiling/transformer_test.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:transformer_test/utils.dart';
+import 'package:scissors/image_dart_transformer.dart';
+
+main() {
+  var iconSvg = r'''
```

Comment:
This would make it inconsistent with other tests that use "var" (these lines have literally been copied from other test)